### PR TITLE
Fixes #3136 - Adds inline images but inside details/summary markup

### LINKED ISSUE #3136: Add back inline screenshot hidden inside <details> by default
See #3122 for relevant code to tweak.

### PR DESCRIPTION
**Pull request by karlcow**
This PR fixes issue #3136

## Proposed PR background

We had removed the inline images, to avoid too much into your face screenshot. Putting them behind a details/summary element is hiding it. 

Also we had to use directly the `img` element instead of the markdown markup. Because markdown is being escaped inside the `details` element.

### CODE REVIEW

**Comment by magsout** [comment]
@karlcow do you forgot to update this line https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/js/lib/bugform.js#L769? or it's intentional?

File: webcompat/static/js/lib/issue-wizard-bugform.js [hunk 1]
```diff
@@ -1373,7 +1373,12 @@ BugForm.prototype.getDataURIFromPreviewEl = function() {
 */
 BugForm.prototype.addImageURL = function(response) {
   var img_url = response.url;
-  var imageURL = ["[Screenshot](", img_url, ")"].join("");
+  var imageURL = [
+    "<details><summary>View the screenshot</summary>",
+    "<img alt='Screenshot' src='",
```
Comment (magsout) [inline webcompat/static/js/lib/issue-wizard-bugform.js]: typo?

` "Screenshot ` "<img alt='Screenshot' src=",`

Comment (magsout) [reply webcompat/static/js/lib/issue-wizard-bugform.js]: maybe is better using template string, more readable, what do you think?
`<details><summary>View the screenshot</summary>Screenshot</details>`

Comment (karlcow) [reply webcompat/static/js/lib/issue-wizard-bugform.js]: ah no the typo is elsewhere 😝 

```diff
-       "></details>"
+       "'></details>"
```

Comment (karlcow) [reply webcompat/static/js/lib/issue-wizard-bugform.js]: template string seems to be a good idea.

Comment (miketaylr) [reply webcompat/static/js/lib/issue-wizard-bugform.js]: The one reason to not use a template strings (yet) would be cross-browser support: https://caniuse.com/#feat=template-literals.

Comment (magsout) [reply webcompat/static/js/lib/issue-wizard-bugform.js]: @miketaylr ah, yes.. true. even on webcompat? I mean about the stats of browsers version of webcompat user? But yes, maybe not a good idea after all.

**Comment by karlcow** [comment]
> @karlcow do you forgot to update this line https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/js/lib/bugform.js#L769? or it's intentional?

@magsout Are you sure?
https://github.com/webcompat/webcompat.com/pull/3154/files#diff-ce241059c28a13e03bcd0829da8d4eb7L769

File: webcompat/static/js/lib/issue-wizard-bugform.js (hunk 1 above)
Comment (karlcow) [reply webcompat/static/js/lib/issue-wizard-bugform.js]: ok modification done. fixed the quote issue in the 3 instances where it appeared.

**Comment by karlcow** [comment]
@magsout we are good for a re-check.

File: webcompat/static/js/lib/issue-wizard-bugform.js (hunk 1 above)
Comment (miketaylr) [reply webcompat/static/js/lib/issue-wizard-bugform.js]: > even on webcompat? I mean about the stats of browsers version of webcompat user?

I just checked, and IE users are 1.87% of our users (wow!). Not super high, but if we can avoid an error for them until they disappear it seems good.

**Pull request merged**